tests/integration/test_image_filters.py

```diff
@@ -612,6 +612,37 @@ def test_rag_threshold_filter_on_rgb_image(pil_image, expected_image):
     )
 
 
+def _create_rag_mask(pil_image):
+    mask = np.ones((pil_image.size[1], pil_image.size[0]))
+    mask[:100, :] = 0
+    mask[:, :100] = 0
+    mask[-100:, :] = 0
+    mask[:, -100:] = 0
+    return mask
+
+
+@pytest.mark.parametrize(
+    "pil_image, expected_image, mask",
```

Comment:
can you put the expectation at the end?
```
"pil_image, mask, expected_image"
```
Thanks
Just to be compliant with the other tests